rcttc-queries.sql

```diff
@@ -59,20 +59,24 @@ group by c.first_name;
 
 -- Calculate the total revenue per show based on tickets sold.
 
-select name, sum(ticket_price)
-from performance
-group by name;
+select p.name, p.date, p.ticket_price * count(t.ticket_id) as total_revenue
+from performance p
+inner join ticket t on p.performance_id = t.performance_id
+group by p.performance_id;
+
 
 -- Calculate the total revenue per theater based on tickets sold.
 
-select th.name, sum(p.ticket_price)
-from performance p
-left outer join theater th on p.theater_id = th.theater_id
-group by th.name;
+select th.name, p.ticket_price * count(t.ticket_id) as total_revenue
+from theater th
+inner join performance p on th.theater_id = p.theater_id
+inner join ticket t on p.performance_id = t.performance_id
+group by p.performance_id;
```

Comment:
Same here. We want revenue per theater. In your result each theater listed multiple times:
```
10 Pin	357.50
10 Pin	297.00
10 Pin	330.00
10 Pin	300.00
Little Fitz	208.20
Little Fitz	208.45
Little Fitz	240.00
Little Fitz	267.00
Horizon	206.50
Horizon	167.75
Horizon	322.50
Horizon	258.00
```
So, we could do this instead:
```
select th.name, sum(p.ticket_price) as total_revenue
from ticket t
inner join performance p on t.performance_id = p.performance_id
inner join theater th on p.theater_id = th.theater_id
group by th.theater_id;
```
and get:
```
10 Pin	1284.50
Little Fitz	923.65
Horizon	954.75
```
(which, btw, is not quite correct result..., there must have been some over-deletes from the original data, as the expected result is this:
```
-- 10 Pin	1197.60
-- Horizon	1064.50
-- Little Fitz	923.65
```